README.md

```diff
@@ -21,6 +22,26 @@
     + [Building](#building)
     + [Using an Nvidia GPU in the container](#using-an-nvidia-gpu-in-the-container)
 
+# Dedicated Client
+
+## What is a Dedicated Client?
+
+A dedicated client in the context of FIKA and SPT is essentially a separate instance of the game client that runs in a headless mode to host raids. It acts as a "silent player" that hosts the raid, allowing other players to join without one of them having to host the raid on their own machine.
+
+- **Offloading Raid Hosting**: By using a dedicated client, the computational load of hosting a raid is offloaded from the player's machine to the dedicated client. This can improve performance for players, especially those with less powerful hardware.
+- **Consistent Hosting**: Ensures that the raid hosting environment is consistent, as it runs on a dedicated server with stable resources.
+- **Centralized Mod Management**: With ModSync, mods can be managed centrally on the server, ensuring all clients have the correct versions.
```

Comment:
While this is true, this doesn't have much to do with the dedicated client, since the central mod source of truth exists on the SPT backend server. Was there a reason to include this that I'm missing?

Perhaps we can mention something about the dedicated client ensuring standardized raid settings regarding AI, difficulty, and spawning?

Reply:
@zhliau Ah yea. I had written it out with this Docker container in mind. I replaced this bullet point with your suggestion.